### DISCUSSION TITLE
heating curve heat pump

### DISCUSSION THREAD

**Discussion by rickroetenberg** [emsesp/EMS-ESP32#973] [General]
Anybody idea if it would be possible to adjust the heating curve from the heat pump through EMS-ESP32? So the foot point and the end point?

**Comment by bbqkees** [comment]
Can this be done from the app/web interface of the heat pump?

If not then we have no way of finding out how it's implemented.

**Comment by rickroetenberg** [comment]
No, but the base point and max. flow temperature can be changed through EMS-ESP32.

Base point = hc1 min flow temperature
Max flow temperature = hc1 max flow temperature

I think that the end point of the heating point should be the hc1 max flow temperature in EMS-ESP32.

![WhatsApp Image 2021-12-21 at 10 47 38 AM](https://user-images.githubusercontent.com/35991592/146908813-20726a24-98db-4e8f-af2c-55267952bb34.jpeg)
![WhatsApp Image 2021-12-21 at 10 47 19 AM](https://user-images.githubusercontent.com/35991592/146908817-df24a2d0-800a-4ee9-bac2-af5dff5fde07.jpeg)
